front/app/containers/Admin/pages-menu/SectionToggle/index.tsx

```diff
@@ -33,11 +34,16 @@ const SectionToggle = ({
   disabled,
   isLastItem,
   hideToggle = false,
+  name,
 }: Props) => {
   return (
     <Row isLastItem={isLastItem}>
       <Box display="flex" alignItems="center" justifyContent="center">
-        <Box visibility={hideToggle ? 'hidden' : 'visible'} mr="20px">
+        <Box
```

Comment:
I wonder if it would be appropriate to use the new accordion / list item components here?

Reply:
![Screen Shot 2022-07-27 at 17 48 24](https://user-images.githubusercontent.com/3614128/181293266-103f8fe5-fe17-4bf0-854f-0ad5e1915240.png)

The use case is to hide one of the toggles here, from looking at Storybook I don't think the accordion/list item would work